src/partials/main.hbs

```diff
@@ -1,11 +1,7 @@
 <main class="article">
 {{> toolbar}}
   <div class="content">
-  <!-- Google Tag Manager (noscript) -->
-  <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-5FSG7Q"
-  height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
-  <!-- End Google Tag Manager (noscript) -->
 {{> article}}
 {{> toc}}
   </div>
-</main>
+</main>
```

Comment:
I added a blank line at the end of the file to make the HTML output more organized.